ledger/ledger.go

```diff
@@ -111,39 +70,41 @@ func NewTestLedger(
 	am *artifactmanager.LedgerArtifactManager,
 	pm *pulsemanager.PulseManager,
 	jc *jetcoordinator.JetCoordinator,
-	amh *artifactmanager.MessageHandler,
 	ls *localstorage.LocalStorage,
 ) *Ledger {
 	return &Ledger{
-		db:      db,
-		am:      am,
-		pm:      pm,
-		jc:      jc,
-		handler: amh,
-		ls:      ls,
+		db: db,
+		AM: am,
```

Comment:
I really really dislike these two-letter abbreviations (except db), could you do smth with them?